code/headers/int_to_string.hpp

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+namespace r2d2::robos {
```

Comment:
You'll need to refactor this a bit, currently is doesn't fit the style guide. You could create a `detail` namespace. Internal communication uses this a lot as well. This could also go to a .cpp file

Reply:
this isn't there anymore since fixed in https://github.com/R2D2-2019/RobOS/pull/2/commits/87c92d1dc8e85dba1e0b57089281459eaa30efef